docs/docs/gatsby-config.md

```diff
@@ -115,6 +115,8 @@ module.exports = {
 }
 ```
 
+You may need to install the appropriate file transformer and source plugins for Gatsby to pick up the mapping files. This applies to other file types later mentioned in this segment as well.
```

Comment:
```suggestion
You may need to install the appropriate file transformer and source plugins for Gatsby to pick up the mapping files. This applies to other file types later mentioned in this segment as well. For example, to handle YAML files, you would install [`gatsby-transformer-yaml`](/packages/gatsby-transformer-yaml/?=yaml). 

> _Note: Additional source plugins can be found in the [plugin library](/plugins?=gatsby-source)._
```

What do you think about adding some links to common source plugins?

Reply:
Hey @marcysutton, I added a link to the YAML transformer plugin and rephrased a little bit. How do you think it reads now?